<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nomTilt">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Tilts</string>
   </property>
  </widget>
  <widget class="QPushButton" name="droite">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>108</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);</string>
   </property>
   <property name="text">
    <string>Droite</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bas">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);</string>
   </property>
   <property name="text">
    <string>Bas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="haut">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>58</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);</string>
   </property>
   <property name="text">
    <string>Haut</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="jogStep">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>115</y>
     <width>61</width>
     <height>36</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="gauche">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>109</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);</string>
   </property>
   <property name="text">
    <string>Gauche</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroButtonLat">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>241</y>
     <width>40</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);
border-radius:3px</string>
   </property>
   <property name="text">
    <string>Zero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>217</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lat</string>
   </property>
  </widget>
  <widget class="QLabel" name="position_Lat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>71</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Trebuchet MS</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QLabel" name="position_Vert">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>71</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>1234</string>
   </property>
  </widget>
  <widget class="QPushButton" name="zeroButtonVert">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>40</width>
     <height>13</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(145, 145, 145);
border-radius:3px</string>
   </property>
   <property name="text">
    <string>Zero</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
